doc/enterprise/security.rst

```diff
@@ -408,275 +408,7 @@ For more information on logging, see the following:
 Access permissions to audit log files can be set up as to any other Unix file
 system object -- via ``chmod``.
 
-.. _enterprise-iproto-encryption:
 
-Traffic encryption
-------------------
-
-Since version 2.10.0, Tarantool Enterprise Edition has the built-in support for using SSL to encrypt the client-server communications over :ref:`binary connections <box_protocol-iproto_protocol>`,
-that is, between Tarantool instances in a cluster or connecting to an instance via connectors using :doc:`net.box </reference/reference_lua/net_box>`.
-
-Tarantool uses the OpenSSL library that is included in the delivery package.
-Please note that SSL connections use only TLSv1.2.
-
-.. _enterprise-iproto-encryption-config:
-
-Configuration
-~~~~~~~~~~~~~
-
-To configure traffic encryption, you need to set the special :ref:`URI parameters <index-uri-several-params>` for a particular connection.
-The parameters can be set for the following ``box.cfg`` options and ``nex.box`` method:
-
-*   :ref:`box.cfg.listen <cfg_basic-listen>` -- on the server side.
```

Comment:
Shouldn't we add it into the `box.cfg` options description (and `net.box` too)?

Reply:
FYI added missing options to the old box.cfg reference: https://github.com/tarantool/doc/pull/4007